modules/kubernetes-addons/appmesh-controller/main.tf

```diff
@@ -1,6 +1,7 @@
 locals {
-  name      = try(var.helm_config.name, "appmesh-controller")
-  namespace = try(var.helm_config.namespace, "appmesh-system")
+  name                 = try(var.helm_config.name, "appmesh-controller")
+  namespace            = try(var.helm_config.namespace, "appmesh-system")
+  service_account_name = try(var.helm_config.service_account, local.name)
```

Comment:
lets standardize as much as we can on `service_account_name`, where it doesn't break the API compatibility

Reply:
@bryantbiggs I addressed this change by allowing both `service_account` and `service_account_name` for the `adot*` modules.

All the others only allow `service_account_name`.

Reply:
@bryantbiggs please review when you get a chance. I'm hoping to have this merged prior to the next release (pretty please) :D

Reply:
my apologies - I didn't take the proper time to look and mislead you to rename incorrectly. It looks like most of the names used are `service_account` or `kubernetes_service_account` so we should use `service_account` and not `service_account_name`. I think I've caught all the instances, will merge once CI passes green

Reply:
Ah ok, thank you for making the changes here.